autotest.pm

```diff
@@ -345,9 +345,11 @@ sub runalltests {
         if (!$vmloaded && $fullname eq $firsttest) {
             if ($bmwqemu::vars{SKIPTO}) {
                 if ($bmwqemu::vars{TESTDEBUG}) {
+                    testapi::record_info('loading snapshot', 'TESTDEBUG set');
```

Comment:
```suggestion
                    $current_test;->record_resultfile('loading snapshot', 'TESTDEBUG set', result => 'ok');
```

Please don't call the external facing test API from internal functions. This is following "design by contract".